pycue/opencue/wrappers/subscription.py

```diff
@@ -37,6 +36,12 @@ def find(self, name):
             timeout=Cuebot.Timeout)
         return Subscription(response.subscription)
 
+    def get(self, id):
+        response = self.stub.Get(subscription_pb2.SubscriptionGetRequest(
```

Comment:
Nit: break on the first paren where possible, it makes it a little easier to read IMO.

```
response = self.stub.Get(
    subscription_pb2.SubscriptionGetRequest(id=id),
    timeout=Cuebot.Timeout)
```

Reply:
Done.